src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sut calendar--month-length-31">
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), class names should be lowercase and if they consist of multiple words, they should be separated by a hyphen. The class `calendar--start-day-sut` seems to contain a typo. It should be `calendar--start-day-sun` if it represents Sunday.